src/ActionModal/components/SendModal.vue

```diff
@@ -349,10 +347,15 @@ export default {
         : this.amount
       if (
         this.network.startsWith(`terra`) &&
-        this.selectedBalance.denom !== `LUNA`
+        this.selectedBalance.denom !== `LUNA` &&
+        this.chainAppliedFees &&
```

Comment:
do we need to wait until those are loaded?

Reply:
well, yes, otherwise how to calculate the Terra Tax. I needed to add this because otherwise it was erroring as soon as I got to Portfolio

Reply:
ahh alright this logic already waits until chainAppliedFees is loaded? what if it is loaded for another coin/network and then you switch and reopen the SendModal?

Reply:
I just checked and it is firing everytime the user changes the network. But it would be good to check once we have the networks navbar on the left